app/views/user_mailer/batch_subscription_notification.html.erb

```diff
@@ -4,62 +4,31 @@
   <% @creations.each_with_index do |creation, index| %>
 
     <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-    <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
+    <% creation_url = creation.is_a?(Work) ? work_url(this_work) : work_chapter_url(this_work, creation) %>
 
-    <% cache("subscription-email-preface-#{creation.cache_key}") do %>
-      <%= creator_links(this_work) %> posted a
-      <% if creation.is_a?(Work) %>
-        <%= this_work.backdate ? "backdated" : "new" %> work:
-      <% else %>
-        new chapter of <i><b><%= work_link %></b></i> (<%= this_work.word_count %> words):
-      <% end %>
+    <% cache("/v1/subscription-email-preface-#{creation.cache_key}") do %>
```

Comment:
As I've mentioned previously, caching and internationalization don't play well together. So far, the translations have primarily been limited to emails, and the other emails don't use caching anywhere. But obviously this does.

I think you need to make some general decisions about how you want to handle the combination of caching and localization. One way to handle it would be to override the `cache` function so that it always adds `I18n.locale` to the cache key. This would prevent (some) bugs like the ones you've introduced here, at the cost of going a little overboard — it means that even if there's a cache block that doesn't have any localized text in it, you won't be able to share the cache between locales. Also, if you do this it will expire all existing `cache` blocks, which means that the deploy for this change would be very rough on the database — might be good to schedule it for a low period.

The other alternative is to add `I18n.locale` to the cache key of each cache block that contains localized text. This allows for more control over the caching, but forces developers to be aware of whether the views that they're modifying are cached, and whether any of the functions they're using inside the cache block return localized text.

(Another note on internationalization and caching: enabling internationalized emails will generate a bunch of extra work when sending subscriptions, because the info for each work may have to be regenerated an extra time for each extra locale. So you may want to do some optimization on the `batch_subscription_notification` function before you enable locales for everyone. Actually, you should probably also do at least a little optimization before this is deployed, since the `work_info` partial uses 6 more database queries per work than the existing code.)

(Another side note: you don't need to add a version number to the cache key here, since you're changing the contents of the cache block, and this doesn't have `skip_digest: true`.)